src/mediator/ModelManager.java

```diff
@@ -12,11 +13,13 @@
     private Heater heater;
     private PropertyChangeSupport property;
     private TemperatureList temperatureList;
+    private Limits limits;
 
     public ModelManager()
     {
         temperatureList = new TemperatureList();
         property = new PropertyChangeSupport(this);
+        limits = new Limits(0,50);
```

Comment:
Maybe a more realistic limit
what is critical temperatures in your room?

Reply:
added